server/src/index.js

```diff
@@ -10,17 +10,38 @@ app.use(express.static("static"));
 app.use(bodyParser.json());
 
 app.get("/beers", async (req, res) => {
-  const beers = await beerService.getBeers();
-  res.send({
-    beers
-  });
+  // TODO: Use this try/catch pattern with the other handlers
```

Comment:
Please remove this comment

Reply:
Comment removed, will try to do this in future.